cmd/queue/main.go

```diff
@@ -104,22 +106,27 @@ func connectStatSink() {
 	autoscalerEndpoint := fmt.Sprintf("ws://%s.%s.svc.cluster.local:%s",
 		servingAutoscaler, system.Namespace, servingAutoscalerPort)
 	logger.Infof("Connecting to autoscaler at %s.", autoscalerEndpoint)
-	for {
-		// TODO: use exponential backoff here
-		time.Sleep(time.Second)
 
+	wait.ExponentialBackoff(wait.Backoff{
+		Duration: 100 * time.Millisecond,
```

Comment:
Probably want to add `Jitter` here to avoid thundering herds when revisions are scaled up quickly.